src/net_processing.cpp

```diff
@@ -4085,6 +4138,11 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
     }
 
     if (msg_type == NetMsgType::FEEFILTER) {
+        if (peer->m_disable_tx) {
```

Comment:
If you have to retouch the code, you can increase the clarity of the logs by precising which protocol violation has been hit, in the occurrence a forbidden sending after a "disable tx received" (as some logs are actually precising). Nice for the node operator.